src/pages/transactions/_components/DfTx/DfTxPoolSwap.tsx

```diff
@@ -5,35 +5,67 @@ import { fromScript } from "@defichain/jellyfish-address";
 import { useNetwork } from "@contexts/NetworkContext";
 import { TokenSymbol } from "@components/commons/token/TokenSymbol";
 import { AddressLink } from "@components/commons/link/AddressLink";
+import { Transaction } from "@defichain/whale-api-client/dist/api/transactions";
+import { useEffect, useState } from "react";
+import { useWhaleApiClient } from "@contexts/WhaleContext";
+import NumberFormat from "react-number-format";
 import { DfTxHeader } from "./DfTxHeader";
 
 interface DfTxPoolSwapProps {
   dftx: DfTx<PoolSwap>;
+  transaction?: Transaction;
 }
 
 export function DfTxPoolSwap(props: DfTxPoolSwapProps): JSX.Element {
   const network = useNetwork().name;
+  const { transaction, dftx } = props;
```

Comment:
You can move this on the function call instead of putting it here

Reply:
Will update it